distro/docker-compose/Readme.md

```diff
@@ -13,50 +13,95 @@ Here is the port mapping:
+Apicurio URL: 192.168.1.231:8093
+Microcks URL: 192.168.1.231:8900
 
-After this step you will be able to run our script, the argument with your IP is mandatory:
 ```
-setup.ps1 192.168.0.1
+
+Please copy these values somewhere where you can find them easily!
+
+### Script based setup
+
+If you're using NIX based OS, you can run the setup script without the docker wrapper. The only dependency is "util-linux" package which contains a tool called uuidgen.
+
+```
+./setup.sh {IP_OF_YOUR_HOST} {DATABASE_TYPE}
 ```
 
 ## Environment customisation
 
-After the successfull run of the setup script, a file called `.env` will appear. This file contains the customisable properties of the environment. Every property is already filled in, so this is only for customisation. You can set your passwords, URL's, and the versions of the components of Apicurio-Studio. The default version is the `latest-release` tagged container from dockerhub, but you can change this as you want. In the near future we will generate the passwords and the Microcks client-secret in the setup process dynamically.
+After the successfull run of the setup script, a file called `.env` will appear. This file contains the customisable properties of the environment. Every property is already filled in, so this is only for customization. You can set your passwords, URL's, and the versions of the components of Apicurio-Studio. The default version is the `latest-release` tagged container from dockerhub, but you can change this as you want.
+
+The passwords for DBs, KeyCloak, and the uuid of the microcks-service-account is generated dynamically with every run of the setup script.
 
 If you want to change these settings (or the provided KeyCloak configuration) after you already started the stack, you have to remove the already existing docker volumes. The easiest way is to stop your running compose stack, and prune your volumes:
 
 ```
 docker system prune --volumes
 ```
 
+A simple "reset" script is also included, it will remove the generated config files for you.
+
+```
+./reset_env.sh
+```
+
 ## Starting the environment
 
-When your configs are generated, you can start the whole stack with this command:
+When your configs are generated, you can start the whole stack with these commands:
 ```
 docker-compose -f docker-compose.keycloak.yml build
-docker-compose -f docker-compose.keycloak.yml -f docker-compose.microcks.yml -f docker-compose.apicurio.yml up
+
+If you generated a config for MySQL:
+docker-compose -f docker-compose.keycloak.yml -f docker-compose.microcks.yml -f docker-compose.apicurio.yml -f docker-compose-as-mysql.yml up
+
+If you generated a config for PostgreSQL:
+docker-compose -f docker-compose.keycloak.yml -f docker-compose.microcks.yml -f docker-compose.apicurio.yml -f docker-compose-as-postgre.yml up
+
 ```
 
+Please do not mix up those commands! If you want to switch between databases, you have to clear the already existing volumes and configs.
+
```

Comment:
Can you provide the commands to do that?  Or perhaps include that in the reset script?

Reply:
done :)